docs/swupd.1.rst

```diff
@@ -352,12 +376,6 @@ SUBCOMMANDS
 
             Matches nothing, because paths are never empty.
 
-    - `-i, --install`
```

Comment:
As we discussed, let's keep this here, and add a comment that this is deprecated and os-install should be used.

Reply:
sure thing